frontends/smv_node.h

```diff
@@ -10,30 +11,1276 @@
 #include "exceptions.h"
 #include "smt-switch/smt.h"
 
+using namespace std;
 /* node to help construct syntax tree in the parser */
```

Comment:
Could you expand on this comment a little more after adding the VarType/NodeMtype/Type and element_node? It'd be nice to have some brief explanation here in the code. I think you can just take some bullet points from the slides.